src/main/resources/messages/en.yml

```diff
@@ -269,20 +269,36 @@ MinecraftPlayerDeathMessage:
       Text: ""
       IconUrl: ""
     Timestamp: false
-
-# Minecraft -> Discord achievement/advancement messages
 #
 # Available placeholders:
```

Comment:
This should say "Available placeholders for PlayerAchievement messages:" (and match in other languages, Google translate can be used to translate)

Reply:
Each messages.yml should now have the same line translated